autorest/mocks/helpers.go

```diff
@@ -11,7 +11,7 @@ const (
 	TestAuthorizationHeader = "BEARER SECRETTOKEN"
 
 	// TestBadURL is a malformed URL
-	TestBadURL = ""
+	TestBadURL = "                               "
```

Comment:
Actually, no can do. TestBadURL is a constant (and should be). Go disallows constant initialization from code (that is, the call to strings.Repeat is disallowed).